common.ini.in

```diff
@@ -43,6 +43,8 @@ redis.socket_connect_timeout = 0.1
 redis.socket_timeout = 0.25
 # timeout waiting for an available slot in the connection pool (seconds)
 redis.pool_timeout = 0.5
+# cache keys TTL (seconds). Used by redis' eviction mechanism, not by dogpile
+redis.expiration_time = 86400
```

Comment:
So, you no longer want to use a LRU strategy? From the way we are using the cache, there would be no need to set an expire time. If a value changes, we change the cache key.